.gitignore

```diff
@@ -20,4 +20,6 @@ output
 
 .vscode
 
-*.wasm
\ No newline at end of file
+*.wasm
+
+*.idea
```

Comment:
press F for jetbrains

Reply:
Yeah turns out I'm very bad at rebasing and somehow it kept trying to pickup the idea files, so I had to make it stop somehow ;)